### PR TITLE
[MNTR] fix test dll loading error

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: src/core/Akka.MultiNodeTestRunner.Shared/Sinks/MessageSink.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ public enum MultiNodeTestRunnerMessageType
         };
 
         private const string NodePassStatusRegexString =
-            @"\[(\w){4}(?<node>[0-9]{1,2})(?<role>:\w+)?\]\[(?<time>\d{1,4}[- /.]\d{1,4}[- /.]\d{1,4}\s\d{1,2}:\d{1,2}:\d{1,2}(\s(AM|PM)){0,1})\]\[(?<status>(PASS|FAIL))\]{1}\s(?<test>.*)";
+            @"\[(\w){4}(?<node>[0-9]{1,2})(?<role>:[a-zA-Z0-9-]+)?\]\[(?<time>\d{1,4}[- /.]\d{1,4}[- /.]\d{1,4}\s\d{1,2}:\d{1,2}:\d{1,2}(\s(AM|PM)){0,1})\]\[(?<status>(PASS|FAIL))\]{1}\s(?<test>.*)";
```
Comment (Aaronontheweb) [inline src/core/Akka.MultiNodeTestRunner.Shared/Sinks/MessageSink.cs:92]: Why this change?

Comment (Arkatufus) [reply src/core/Akka.MultiNodeTestRunner.Shared/Sinks/MessageSink.cs:92]: it fails when role contains a number or a dash, some of the test have the role as "node-1"

Comment (Arkatufus) [reply src/core/Akka.MultiNodeTestRunner.Shared/Sinks/MessageSink.cs:92]: Fails as in the regex won't recognize the message type as a pass/fail message

Comment (Aaronontheweb) [reply src/core/Akka.MultiNodeTestRunner.Shared/Sinks/MessageSink.cs:92]: Looks like the MNTR internals suite is failing due to this now

**Comment by Aaronontheweb** [comment]
LGTM
